website/docs/r/record.html.markdown

```diff
@@ -168,6 +166,10 @@ The following arguments are supported:
   `us_state` should be comma separated strings, and changes in ordering will not
   lead to terraform detecting a change.
 
+Note: regions **must** be sorted lexically by their "name" field in the `.tf`
+file, otherwise Terraform will detect changes to the record when none actually
```

Comment:
I'd suggest changing `name field` to `name argument` and `.tf file` to `Terrform configuration` to stick with the nomenclature of the project.

Reply:
wilco